pkg/state/temp_test.go

```diff
@@ -37,39 +37,39 @@ func TestGenerateID(t *testing.T) {
 	run(testcase{
 		subject: "baseline",
 		release: ReleaseSpec{Name: "foo", Chart: "incubator/raw"},
-		want:    "foo-values-67b55dc69b",
+		want:    "foo-values-779795898b",
```

Comment:
@mumoshu I'm not 100% on if I should be doing this since the outcome of the suffix has changed or if I should be striving to make the tests have identical suffix?  I'm guessing with the introduce of the additional helmDefaults it required this change?

Reply:
> I'm guessing with the introduce of the additional helmDefaults it required this change?

@jwitko I believe so, and it's behaving as expected in terms of implementation of generateIValuesID func so you're all good 👍